src/dmd/frontend.d

```diff
@@ -32,12 +32,13 @@ void initDMD()
     import dmd.expression : Expression;
     import dmd.globals : global;
     import dmd.id : Id;
-    import dmd.mars : addDefaultVersionIdentifiers;
+    import dmd.mars : setTarget, addDefaultVersionIdentifiers;
```

Comment:
This looks like a bug. Isn't set target a private function?

Reply:
not anymore!